modules/spo-log-enricher.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-advanced.adoc
+
+:_content-type: PROCEDURE
+[id="spo-log-enricher_{context}"]
+= Using the log enricher
+
+The Security Profiles Operator contains a log enrichment feature, which is disabled by default. The log enricher container runs with `privileged` permissions to read the audit logs from the local node. The log enricher runs within the host PID namespace, `hostPID`.
+
+[IMPORTANT]
+====
+The log enricher must be allowed to read the host processes.
+====
+
+One of the following requirements to the Kubernetes node must be fulfilled to use the log enrichment feature:
```

Comment:
I think for OCP, we can remove this requirement list. It's valid for upstream, but for OCP we enable `auditd` by default and I'm not sure you can even enable syslog.